src/languages/es.js

```diff
@@ -2155,9 +2155,11 @@ export default {
     demos: {
         saastr: {
             signInWelcome: '¡Bienvenido a SaaStr! Entra y empieza a establecer contactos.',
+            heroBody: 'Utiliza New Expensify para estar al día de los eventos, establecer contactos, charlar en las redes sociales, ¡y para que te devuelvan el dinero de la comida!',
         },
         sbe: {
             signInWelcome: '¡Bienvenido a Small Business Expo! Recupera el dinero de tu viaje.',
+            heroBody: 'Utiliza New Expensify para estar al día de los eventos, establecer contactos, charlar en las redes sociales y para que te paguen el viaje de ida y vuelta a la feria.',
```

Comment:
Still need translations to be approved, i requested here: https://expensify.slack.com/archives/C21FRDWCV/p1693398433886809

Reply:
done!